parser.js

```diff
@@ -199,6 +199,14 @@ Parser.propertiesParser = function(filename, content) {
   return PPARSER.parse(content, { namespaces: true, variables: true, sections: true });
 };
 
+Parser.numberParser = function(filename, content) {
+    return Number(content);
```

Comment:
What value will be in result config in case if we have empty env variable?
```
MY_VAR=
```

```
"number": {
            "__name": "MY_VAR",
            "__format": "number"
        }
```
And in default.js
```
module.exports = {
  number: 1
}
```

Reply:
Quirky JavaScript:

```
> Number(undefined)
NaN
> Number("")
0
> Number(null)
0
> Number("1")
1
> Number("0.5")
0.5
```

I'd suggest an extra condition here to account for the `undefined` case, since this is what will happen if the env var is not set:

``` js
Parser.numberParser = function(filename, content) {
  const val = Number(content);
  return Number.isNaN(val) ? undefined : val;
};
```